cli/internal/cloudcmd/tfupgrade.go

```diff
@@ -16,20 +17,30 @@ import (
 	"github.com/edgelesssys/constellation/v2/internal/file"
 )
 
-// planUpgrade prepares a workspace and plans the possible Terraform migrations.
+// planApply prepares a workspace and plans the possible Terraform actions.
+// This will either create a new workspace or update an existing one.
 // In case of possible migrations, the diff is written to outWriter and this function returns true.
-func planUpgrade(
-	ctx context.Context, tfClient tfUpgradePlanner, fileHandler file.Handler,
+func planApply(
```

Comment:
```suggestion
func plan(
```
opinion: original name made me wonder if this also ran `apply`.